src/historywork/GzipFileWork.cpp

```diff
@@ -10,7 +10,8 @@ namespace stellar
 
 GzipFileWork::GzipFileWork(Application& app, std::string const& filenameNoGz,
                            bool keepExisting)
-    : RunCommandWork(app, std::string("gzip-file ") + filenameNoGz)
+    : RunCommandWork(app, std::string("gzip-file ") + filenameNoGz,
```

Comment:
I am not sure I would not retry gzip: the only reason this would fail is because it runs out of disk space or it cannot spawn processes (we've seen both).
Retrying just that step seems desirable under those specific failure modes. Ideally it would even retry forever but there is a chance of some corruption (like for some reason the files it's trying to zip got deleted somehow) and we may be better off resetting the whole flow